updates/ryan3.sql

```diff
@@ -0,0 +1,48 @@
+
+Create Table selektion_beziehung(ID int Not NULL AUTO_INCREMENT, Bezeichnung varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
```

Comment:
All SQL Keywords should be written in uppercase

CREATE TABLE INT NOT NULL VARCHAR

Reply:
done